.eslintrc

```
@@ -0,0 +1,31 @@
+{
+  "extends": [
+    "airbnb",
+    "airbnb-typescript",
+    "airbnb/hooks",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking"
+  ],
+
+  "parserOptions": {
+      "project": ["./tsconfig.json"]
+  },
+
+  "rules": {
+    "@typescript-eslint/indent": "off",
+    "@typescript-eslint/no-var-requires": 0,
+    "semi": "error",
+    "arrow-body-style": "off",
+    "react/function-component-definition": [
+      2,
+      {
+        "namedComponents": "arrow-function",
+        "unnamedComponents": "arrow-function"
+      }
+    ]
```

```suggestion
    ],
    "no-plusplus": [
        "error",
        {
          "allowForLoopAfterthoughts": true
        }
      ],
      "no-unused-vars": "off",
      "@typescript-eslint/no-unused-vars": [
        "off"
      ]
```


1. Я понимаю, что это может ухудшать читаемость, но расписывать в каждом for "i = i + 1" категорически лень)
2. Тайпскрипт и так дает ошибку на unused variables, а две ошибки на каждый случай не надо

---

не очень понял оба случая.. в первом модуле у меня без этого правила i++ в циклах .ts файлов и ничего.. =)
во втором ты предлагаешь заменить @typescript-eslint/no-unused-vars на no-unused-vars?

Возьму пока на заметку ;-)

---

По поводу первого – окей, не уверен, поглядим)

Второе – предлагаю просто выпилить это правило из eslint, ибо его и так будет проверять tsc (по крайней мере vs code стабильно ругается)